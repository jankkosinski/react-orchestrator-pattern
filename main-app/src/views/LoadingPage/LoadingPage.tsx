import React from 'react';
import './styles.css';

export const LoadingPage = () => {
  return <div id='Loading' className='container' />;
};
