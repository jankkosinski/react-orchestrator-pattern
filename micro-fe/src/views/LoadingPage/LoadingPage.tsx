import React from 'react';
import styles from './loadingpage.module.scss';

export const LoadingPage = () => {
  return <div className={styles.container} />;
};
