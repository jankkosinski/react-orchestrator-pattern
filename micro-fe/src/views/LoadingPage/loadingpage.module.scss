@import '../../scss/colors';

.container {
  display: flex;
  flex-direction: row;
  justify-content: center;
  align-items: center;
  height: 100vh;
  width: 100vw;
  color: $secondary;
  background-color: $primary;
}
