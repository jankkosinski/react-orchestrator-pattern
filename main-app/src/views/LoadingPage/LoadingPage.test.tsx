import { render } from '@testing-library/react';
import React from 'react';
import { LoadingPage } from './LoadingPage';

describe('LoadingPage', () => {
  it('Renders the LoadingPage component', () => {
    const { container } = render(<LoadingPage />);
    expect(container).toMatchSnapshot();
  });
});
