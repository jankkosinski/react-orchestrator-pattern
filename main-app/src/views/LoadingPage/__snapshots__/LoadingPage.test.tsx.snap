// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`LoadingPage > Renders the LoadingPage component 1`] = `
<div>
  <div
    class="container"
    id="Loading"
  >
    LoadingPage
  </div>
</div>
`;
